tt_metal/impl/buffers/buffer_constants.hpp

```diff
@@ -31,6 +32,39 @@ enum class BufferType {
     TRACE,
 };
 
+inline std::ostream& operator<<(std::ostream& os, TensorMemoryLayout layout) {
```

Comment:
Can you please move this into `.cpp`?

Reply:
Including all other newly added `operator<<`

Reply:
I'll do. I only moved where .cpp existed so far, but it makes sense. Thanks